### DISCUSSION TITLE
Отчёты в разные телеграмм-боты для разных аккаунтов

### DISCUSSION THREAD

**Discussion by pashva11** [masterking32/MasterHamsterKombatBot#62] [Q&A] [answered]
У меня есть аккаунт в игре Hamster Kombat.
И у моей жены есть аккаунт в игре Hamster Kombat.
Я добавил оба наших аккаунта в MasterHamsterKombatBot.

1. Можно сделать чтобы можно было настроить так: сообщения о моем аккаунте приходят мне в телеграмм, а сообщения о событиях в аккаунте жены приходят ей в телеграмм? Мне хотелось бы получать сообщения о моём аккаунте и не получать о её аккаунте. А жене хотелось бы получать о своём аккаунте сообщения, а о моем не получать.

2. Можно ли сделать так, чтобы сообщения в телеграмм приходили не так часто, например раз в один час или раз в три часа?

**Comment by AmirhN1382** [comment, marked as answer]
Hey,
1- Yes it's possible, put a different chat id at this [line](https://github.com/masterking32/MasterHamsterKombatBot/blob/main/main.py#L63C10-L63C26)
2- No, but you can turn off some logs so you only get important data or mute the bot if it annoys you.

**Comment by pashva11** [comment]
Спасибо! Разобрался! Всё работает!!!
